src/styles/blocks/header.scss

```diff
@@ -0,0 +1,88 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  &__nav {
+    margin: 0;
+    padding: 0;
+    justify-content: space-between;
+    align-items: center;
+    text-transform: uppercase;
+    list-style: none;
+
+  }
+
+  &__item {
+    margin: 0 10px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+  }
+
+  &__link {
+    display: flex;
+    text-decoration: none;
+    color: #060b35;
+    padding: 23px 0;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 12px;
+
+    &:hover {
+      color: rgba(0, 172, 220, 1);
+      transition: 0.3s;
+    }
+  }
+
+}
+
+.is-active {
```

Comment:
please use the modifier `header__link--active' in this case you can write this styles after &__link values. Class `is-active` would be useful only for tests.